en,zh: optimize docs for TiKV upgrade configuration (#1605)
pingcap/docs-tidb-operator: en/configure-a-tidb-cluster.md

### Configure graceful upgrade for TiKV cluster

> During TiKV upgrade, TiDB Operator evicts all Region leaders from TiKV Pod before restarting TiKV Pod. Only after the eviction is completed (which means the number of Region leaders on TiKV Pod drops to 0) or the eviction exceeds the specified timeout (1500 minutes by default), TiKV Pod is restarted. Besides, If the TiKV replicas is less than 2, TiDB Operator will force an upgrade without waiting for the timeout.

Reviewer: ```suggestion
During TiKV upgrade, TiDB Operator evicts all Region leaders from TiKV Pod before restarting TiKV Pod. Only after the eviction is completed (which means the number of Region leaders on TiKV Pod drops to 0) or the eviction exceeds the specified timeout (1500 minutes by default), TiKV Pod is restarted. If TiKV has fewer than 2 replicas, TiDB Operator forces an upgrade without waiting for the timeout.
```
